content/en/docs/changelog/_index.md

```diff
@@ -3,6 +3,13 @@ title = "Changelog"
 description = "Kubeflow on Google Cloud Changelog"
 weight = 300
 +++
+## 1.8.0
+
+[Release notes](https://github.com/GoogleCloudPlatform/kubeflow-distribution/releases/tag/v1.8.0)
+
+### Changes:
+
+- 🔼 Upgraded upstream `Manifests` to `v1.8.0`.
```

Comment:
Was it the only change?

Reply:
Yes, that's only meaningful change worth highligh in release note (there're two more merged changed which are considered chore)